src/internal/server/PrintLeftoverLabelHandler.go

```diff
@@ -0,0 +1,88 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"main/internal/pdf"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type PrintLabelRequestBody struct {
+	LabelText string `json:"labelText"`
+	Quantity  int    `json:"quantity"`
+}
+
+func PrintLeftoverLabelHandleFunction(w http.ResponseWriter, r *http.Request) {
+	/* --FAIL FAST-- */
+
+	if r.Method != "POST" {
+		msg := "This endpoint only supports POST requests"
+		http.Error(w, msg, http.StatusBadRequest)
+		return
+	}
+
+	// ensure Content-Type "application/json"
+	ct := r.Header.Get("Content-Type")
+	if ct != "" {
+		mediaType := strings.ToLower(strings.TrimSpace(strings.Split(ct, ";")[0]))
+		if mediaType != "application/json" {
+			msg := "Content-Type header is not application/json"
```

Comment:
Just to make debug easier for the caller I like to add also content-type we received in the request:

e.g.

```
Content-Type header is not application/json. Received: text/html
```